smithy-aws-protocol-tests/model/awsJson1_0/required.smithy

```diff
@@ -0,0 +1,125 @@
+$version: "2.0"
+
+namespace aws.protocoltests.json10
+
+use aws.protocols#awsJson1_0
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+apply OperationWithDefaults @httpResponseTests([
```

Comment:
Maybe there could be cases where it's required and default and the client correctly uses the default value when error correcting

Reply:
Would be good to have nested cases too where a service responds with a top-level member that's populate with `{}` but has lots of error corrected defaults. Maybe mix that in with things that are serialized too.

```
{"topLevel": {"nested": {"present": 100}}
```